internal/test/issues/issue-1039/types.gen.go

```diff
@@ -0,0 +1,149 @@
+// Package issue1039 provides primitives to interact with the openapi HTTP API.
+//
+// Code generated by github.com/deepmap/oapi-codegen/v2 version v2.0.0-00010101000000-000000000000 DO NOT EDIT.
+package issue1039
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/base64"
+	"fmt"
+	"net/url"
+	"path"
+	"strings"
+
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/oapi-codegen/nullable"
+)
+
+// BreakoutGroup A breakout group which an user will be part of
+type BreakoutGroup struct {
+	// Name Name of the breakout group
+	Name *string `json:"name,omitempty"`
+}
+
+// Organization Org to which the user belong to.
+type Organization struct {
+	AliasName nullable.Nullable[string] `json:"alias_name,omitempty"`
+
+	// Name Name of the organization.
+	Name *string `json:"name,omitempty"`
+}
+
+// PatchRequest A request to patch an existing user object.
+type PatchRequest struct {
+	// BreakoutGroup A breakout group which an user will be part of
+	BreakoutGroup nullable.Nullable[BreakoutGroup] `json:"breakout_group"`
+
+	// HomeCountry Home country of the user.
+	HomeCountry *HomeCountry `json:"home_country,omitempty"`
```

Comment:
At first glance of the tests, I wasn't sure why this was `nil`

It may be worth us tweaking the spec to make the property names clearer what they're testing - i.e. this could be `OptionalButNotNullableProperty` then it's super clear what's being tested (and why it's being checked for `nil` instead of expecting a `nullable.Nullable`)

Reply:
This a very good point. Let me do that